vlib/x/websocket/websocket_test.v

```
@@ -24,20 +24,19 @@ fn start_server() ? {
 		return true
 	}) ?
 	s.on_message(fn (mut ws websocket.Client, msg &websocket.Message) ? {
-		// payload := if msg.payload.len == 0 { '' } else { string(msg.payload, msg.payload.len) }
-		// println('server client ($ws.id) got message: opcode: $msg.opcode, payload: $payload')
 		ws.write(msg.payload, msg.opcode) or {
 			panic(err)
 		}
 	})
 	s.on_close(fn (mut ws websocket.Client, code int, reason string) ? {
-		// println('client ($ws.id) closed connection')
+		// not used
 	})
 	s.listen() or {
```

```
s.listen()?
```

---

`s.listen()?` is not the same as `s.listen() or {}`; the first will propagate the error to the caller, the second will ignore the error completely; I do not know what is appropriate for this context though.

---

the upper one returns an optional as well. if not with `?` then do `or {}`

---

Yea I will go through the tests when I move it to net module from x. I want to refactor it a bit more. I take this comment with me doing this. Think we will could potentially fail without failing tests looking at the test as written now.